packages/utilities/psammead-assets/src/svgs/navigationIcons.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MenuIcon = styled.svg`
+  color: #fff;
+  fill: currentColor;
+`;
+
+const navigationIcons = {
+  hamburger: (
+    <MenuIcon width={44} height={44}>
```

Comment:
Width and height properties should have units associated with them.
```suggestion
    <MenuIcon width="44px" height="44px">
```